src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -403,7 +403,9 @@ public sealed partial class FileSystem
         public static void Input(int FileNumber, ref object Value) { }
         public static void Input(int FileNumber, ref float Value) { }
         public static void Input(int FileNumber, ref string Value) { }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("ios")]
```

Comment:
FYI: @MaximLipnin this was missed on your PR. Let me know if it makes sense.

Reply:
I'm not sure if it's related to `Lock/Unlock` methods which cause PNSE on iOS/tvOS and were annotated in my PR, but it seems that it makes sense to annotate this particular method as well, thank you.